src/components/frame/HamburgerDropdown/HamburgerDropdownPopover.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+// import { defineMessages } from 'react-intl';
+
+import DropdownMenu from '~shared/DropdownMenu';
+import {
+  ColonySection,
+  CreateColonySection,
+  HelperSection,
+  MetaSection,
+  UserSection,
+} from '~shared/PopoverSection';
+import { useAppContext, useColonyContext } from '~hooks';
+
+import styles from './HamburgerDropdownPopover.css';
+
+const displayName = 'frame.HamburgerDropdown.HamburgerDropdownPopover';
+
+// const MSG = defineMessages({
+//   buyTokens: {
+//     id: `${displayName}.buyTokens`,
+//     defaultMessage: 'Buy Tokens',
+//   },
+// });
+
+interface Props {
+  closePopover: () => void;
+}
+
+const HamburgerDropdownPopover = ({ closePopover }: Props) => {
+  const { colony } = useColonyContext();
+  const { user, wallet } = useAppContext();
+
+  return (
+    <div className={styles.menu}>
+      <DropdownMenu onClick={closePopover}>
+        {user?.name && colony?.name && <ColonySection />}
+        <UserSection />
+        <CreateColonySection />
```

Comment:
Create colony should only show up if the user can interact with the network _(see the hook with the same name)_